### PR TITLE
Add Contact Information step to organizer form

### PR DESCRIPTION
**Pull request by Anahkiasen**
### Added

- Add Contact Information step to organizer form

---

Ticket: https://jira.uitdatabank.be/browse/III-4494

### CODE REVIEW

File: src/pages/organizers/create/OrganizerForm.tsx [hunk 1]
```diff
@@ -55,10 +64,28 @@ const typeAndThemeStepConfiguration: StepsConfiguration<'nameAndUrl'> = {
...
-const OrganizerForm = () => {
+const OrganizerForm = (props) => {
+  const { t } = useTranslation();
   const { form } = useParseStepConfiguration(configurations);
+  const { query, push, pathname, reload } = useRouter();
+  const organizerId = query?.organizerId;
+
+  const scope = 'organizers';
+  const createOrganizer = useCreateOrganizerMutation({
+    onSuccess: () => push(`/${scope}/${organizerId}/preview`),
```
Comment (brampauwelyn) [inline src/pages/organizers/create/OrganizerForm.tsx:85]: Should probably redirect to edit?
```suggestion
    onSuccess: () => push(`/${scope}/${organizerId}/edit`),
```

File: src/pages/organizers/create/OrganizerForm.tsx [hunk 2]
```diff
@@ -72,6 +99,20 @@ const OrganizerForm = () => {
...
         />
       </Page.Content>
+      <Page.Footer>
+        <Button
+          variant={ButtonVariants.SUCCESS}
+          onClick={async () =>
+            await createOrganizer.mutateAsync({
+              name: form.getValues().nameAndUrl.name,
+              url: form.getValues().nameAndUrl.url,
+            })
```
Comment (brampauwelyn) [inline src/pages/organizers/create/OrganizerForm.tsx:109]: mainLanguage is mandatory
```suggestion
            await createOrganizer.mutateAsync({
              name: form.getValues().nameAndUrl.name,
              url: form.getValues().nameAndUrl.url,
              mainLanguage: i18n.language
            })
```

File: src/pages/organizers/create/OrganizerForm.tsx [hunk 3]
```diff
@@ -72,6 +99,20 @@ const OrganizerForm = () => {
           form={form}
         />
       </Page.Content>
+      <Page.Footer>
+        <Button
```
Comment (brampauwelyn) [inline src/pages/organizers/create/OrganizerForm.tsx:103]: Good idea to add the button in the Footer 👍

File: src/pages/organizers/create/OrganizerForm.tsx [hunk 4]
```diff
@@ -72,6 +99,20 @@ const OrganizerForm = () => {
...
+          variant={ButtonVariants.SUCCESS}
+          onClick={async () =>
```
Comment (brampauwelyn) [inline src/pages/organizers/create/OrganizerForm.tsx:105]: I would use react-form-hook's `handleSubmit` here so it triggers validation etc?

Comment (Anahkiasen) [reply src/pages/organizers/create/OrganizerForm.tsx:105]: Yeah this is just a draft PR so I stopped there but definitely not final iteration 😅

**Pull request merged**